dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -93,19 +93,71 @@ For the data types above:
 ##############
 
 - Change the axes that are displayed by selecting the relevant ``X`` and ``Y`` axes in the top left. This is more interesting for higher dimension data.
-- For the ``SXD_MD_nonortho`` workspace, the non-orthogonal view button (see below) should be enabled - clicking this should also turn on grid lines. When viewing the H and K axes you should see the gridlines are no longer perpendicular to each other.
+
+4. Non-orthogonal view
+######################
+
+A. For the ``SXD_MD_nonortho`` workspace, the non-orthogonal view button (see below) should be enabled - clicking this should also turn on grid lines. When viewing the H and K axes you should see the gridlines are no longer perpendicular to each other.
 
 .. figure:: ../../../../docs/source/images/wb-sliceviewer51-nonorthobutton.png
    :class: screenshot
    :align: center
 
-4. Cursor Tracking
+B. This tests that the sliceviewer gets the correct basis vectors for an ``MDHisto`` object from a non-axis aligned cut.
+
+.. code-block:: python
+
+    ws = CreateMDWorkspace(Dimensions='3', Extents='-3,3,-3,3,-3,3',
+                       Names='H,K,L', Units='r.l.u.,r.l.u.,r.l.u.',
+                       Frames='HKL,HKL,HKL',
+                       SplitInto='2', SplitThreshold='10')
+    expt_info = CreateSampleWorkspace()
+    ws.addExperimentInfo(expt_info)
+    SetUB(ws, 1,1,2,90,90,120)
+    BinMD(InputWorkspace=ws, AxisAligned=False,
+        BasisVector0='[00L],r.l.u.,0,0,1',
+        BasisVector1='[HH0],r.l.u.,1,1,0',
+        BasisVector2='[-HH0],r.l.u.,-1,1,0',
+        OutputExtents='-4,4,-4,4,-0.25,0.25',
+        OutputBins='101,101,1', OutputWorkspace='ws_slice', NormalizeBasisVectors=False)
+
+- Run the above code and open ``ws_slice`` in the sliceviewer.
+- Check that non-orthogonal view is enabled on opening - however when clicked the gridlines are orthogonal  (in this case 110 is orthogonal to 001).
+
+C. Check that changing the properties of the workspace that governs the support for the non-orthogonal transform closes the sliceviewer window.
+- Open ``ws`` from the above test script in the sliceviewer
+
+- Run ``ClearUB(ws)`` (it should close sliceviewer with warning ``property supports_nonorthogonal_axes is different...``
```

Comment:
```suggestion
C. Check that changing the properties of the workspace that governs the support for the non-orthogonal transform closes the sliceviewer window.

- Open ``ws`` from the above test script in the sliceviewer

- Run ``ClearUB(ws)`` (it should close sliceviewer with warning ``property supports_nonorthogonal_axes is different...``)
```

Sorry, so very close, but I need a newline before the "Open ws" bullet point and an end bracket.

Reply:
OK thanks - will do this and rebase -i to rejig history a bit

Reply:
I'm all happy with the docs now so I'll leave the full review to Martyn!